modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -90,7 +106,56 @@ roleRef:
 $ oc apply -n <target_namespace> -f <dpdk_roles>.yaml
 ----
 
-. Create a `ConfigMap` manifest that contains the input parameters for the checkup. The config map also stores the results of the checkup.
+. Create a `SecurityContextConstraints` manifest for the traffic generator pod:
++
+.Example security context constraints manifest
+[source,yaml]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: dpdk-checkup-traffic-gen
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+- IPC_LOCK
+- NET_ADMIN
+- NET_RAW
+- SYS_RESOURCE
+defaultAddCapabilities: null
+fsGroup:
+  type: RunAsAny
+groups: []
+readOnlyRootFilesystem: false
+requiredDropCapabilities: null
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+seccompProfiles:
+- runtime/default
+- unconfined
+supplementalGroups:
+  type: RunAsAny
+users:
+- system:serviceaccount:dpdk-checkup-ns:dpdk-checkup-traffic-gen-sa
+volumes:
+- hostPath
+----
+
+. Apply the `SecurityContextConstraints` manifest:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <dpdk_scc>.yaml
```

Comment:
SCC is not a namespaced object, so the namespace specification could be dropped:
```suggestion
$ oc apply -f <dpdk_scc>.yaml
```